pkgs/tools/networking/grpc_cli/default.nix

```diff
@@ -0,0 +1,32 @@
+{ lib, stdenv, fetchFromGitHub, automake, cmake, autoconf, curl, numactl }:
+
+stdenv.mkDerivation rec {
+  pname = "grpc_cli";
+  version = "1.46.6";
+
+  buildInputs = [ automake cmake autoconf curl numactl ];
+  src = fetchFromGitHub {
+    owner = "grpc";
+    repo = "grpc";
+    rev = "18dda3c586b2607d8daead6b97922e59d867bb7d";
+    sha256 = "sha256-m55FKwqlAQ2Ge7y4RMFO4W1onu0NYARWrOygChJMxng=";
+    fetchSubmodules = true;
+  };
+  configurePhase = "true";
```

Comment:
Isn't this true by default?

Reply:
I believe by default it calls `./configure`, based on https://nixos.org/manual/nixpkgs/stable/#ssec-configure-phase. And I want this phase to be a no-op, so I just run `true`.